node/src/components/storage/tests.rs

```diff
@@ -378,15 +381,25 @@ fn get_highest_complete_block_header(
     response
 }
 
-/// Stores a block in a storage component.
+/// Stores a block in a storage component and mark it as complete.
```

Comment:
```suggestion
/// Stores a block in a storage component and marks it as complete.
```

Reply:
fixed in https://github.com/casper-network/casper-node/pull/3487/commits/9d689b2485d2024b664e12bbca632db8200d36a8